src/dtclient/kubernetes_settings.go

```diff
@@ -133,6 +118,57 @@ func (dtc *dynatraceClient) CreateOrUpdateKubernetesSetting(clusterLabel, kubeSy
 	return resDataJson[0].ObjectId, nil
 }
 
+func (dtc *dynatraceClient) getKubernetesSettingBody(clusterLabel, kubeSystemUUID, scope string, schemaId string, isK8sHierarchicalMonitoringSettings bool) []postKubernetesSettingsBody {
+	currentSchemaVersion := defaultSchemaVersion
+	if isK8sHierarchicalMonitoringSettings {
+		currentSchemaVersion = hierarchicalMonitoringSettingsSchemaVersion
+	}
+	body := []postKubernetesSettingsBody{
+		{
+			SchemaId:      schemaId,
+			SchemaVersion: currentSchemaVersion,
+			Value: postKubernetesSettings{
+				Enabled:          true,
+				Label:            clusterLabel,
+				ClusterIdEnabled: true,
+				ClusterId:        kubeSystemUUID,
+			},
+		},
+	}
+	if scope != "" {
```

Comment:
can we always set this for the `body` as the default value is also an empty string?